angular.json

```diff
@@ -27,7 +27,9 @@
             "styles": [
               "./node_modules/bootstrap/dist/css/bootstrap.min.css",
               "src/styles.scss",
-              "src/assets/scss/main.scss"
+              "src/assets/scss/main.scss",
+							"node_modules/bootstrap/scss/bootstrap.scss",
```

Comment:
this line is duplicated. Line 28 does the same thing.
 Also, make sure that main.scss is after all the other CSS files to prevent custom syles from being overridden.

Reply:
Thanks, @patrickf949 This has been fixed